### PR TITLE
Bitcoin Price Tracking Python Script

### LINKED ISSUE #789: BITCOIN Price Tracker
Track Bitcoin Price In Real-Time.

## Language

- [ ] Bash
- [ ] Go
- [ ] Javascript
- [ ] Powershell
- [x] Python

### PR DESCRIPTION
**Pull request by pritamp17**
# Description
Track Bitcoin Price In Real-Time.
Fixes #789
## Type of change
- [x] New feature (non-breaking change which adds functionality)
# Checklist:
- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by pritamp17** [comment]
@HarshCasper could you review this PR?

**Review by sohamsshah — CHANGES_REQUESTED** [review]
@pritamp17 This looks good and clean. Good work! 

Would suggest a few changes:
* Rename `BITCOIN-price-tracker` to `Bitcoin_Price_Tracker`
* You can use Python `f-strings` instead of concatenation wherever possible in the code.

File: Python/BITCOIN-price-tracker/tracker.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import requests
+from bs4 import BeautifulSoup
+import time
+# create a function to get price of cryptocurrency
+def get_latest_crypto_price(coin):
```
Comment (HarshCasper) [inline Python/BITCOIN-price-tracker/tracker.py:5]: Put a Docstring

File: Python/BITCOIN-price-tracker/tracker.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    url = 'https://www.google.com/search?q='+(coin)+'price'
+    # make a request to the website
+    HTML = requests.get(url)
+    # Parsse the HTML
+    soup = BeautifulSoup(HTML.text, 'html.parser')
+    # finding the current price
+    texti = soup.find('div', attrs={'class':'BNeawe iBp4i AP7Wnd'}).find({'div':'BNeawe iBp4i AP7Wnd'}).text
+    return texti
+price = get_latest_crypto_price('bitcoin')
+print('BITCOIN price : ' + price)
```
Comment (HarshCasper) [inline Python/BITCOIN-price-tracker/tracker.py:15]: Enclose this in a `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
@pritamp17 

Are you still working on it?

**Comment by vybhav72954** [comment]
@pritamp17 Kindly reply by tonight, else i would be forced to close this PR for no activity.

**Pull request closed without merging**